### DISCUSSION TITLE
Change the position of the tieredmenu bound to splitButton to right

### DISCUSSION THREAD

**Discussion by albertB4work** [primefaces/community#2664] [PrimeNG]
Now, when you open the TieredMenu associated with the SplitButton it is opened to the left of the button, i've tried to modify the styles to put left:100% but the menu is misplaced far-right of the right side of the button. Is there a way to place the dropdown to the right side of the associated button? If not, you would consider a feature for this?

Thanks

Now:

![Captura de pantalla 2024-09-10 a las 10 55 18](https://github.com/user-attachments/assets/a7294766-ea03-4a70-986b-b7e8b708f67e)

Desired:
![image](https://github.com/user-attachments/assets/d2f2de7a-0ee2-4aea-b80f-32a30f1b4b1a)

**Comment by qwadrox** [comment]
Hi @albertB4work!

You can achieve this with CSS. Check out the example I created for you: 
https://stackblitz.com/edit/dhqa1q?file=src%2Findex.html
<img width="312" alt="image" src="https://github.com/user-attachments/assets/42b84ae5-1146-4ad9-9535-4ac9c20e4a6f">
